Test/Platform/Tests/CLR/mscorlib/systemlib/systemlib2/SystemDateTimeTests.cs

```diff
@@ -1941,11 +1944,11 @@ public MFTestResults DateTime_TicksTest52()
             try
             {
                 Log.Comment("Creating a DateTime, getting the Ticks and Verifying");
-                DateTime testDateTime = new System.DateTime(0);
+                DateTime testDateTime = new System.DateTime(635688622030533832);

```

Comment:
Actually, this is not a magic number but is a random number. The goal of the test as I understood is to make sure when you create a DateTime with a tick value, you get the same value from the Ticks property.